### PR TITLE
Recommend SMCAMDProcessor for AMD CPU Temps

### PR DESCRIPTION
**Pull request by 266-750Balloons**
There is a good set of kexts by trulyspinach that allows for Zen CPU temperature monitoring and fan control, AMDRyzenCPUPowerManagement and SMCAMDProcessor (the latter being what enabled the features and the former being the corresponding and dependant SMC plugin), respectively. It is in early development, but I think it is worthy of being added with warnings, as most Hackintoshes are not used in mission critical situations anyway. Thus, I have added information on this kext in ktext.md and AMD/zen.md to inform of this option.
I know these kexts to work well because I have tested them and currently use them on my own machine with the following specifications:
* CPU: AMD Ryzen 5 2600
* GPU: AMD Radeon RX 580
* Motherboard: Gigabyte Aorus AX370 Gaming 5 (No MATs support, 4G encoding enabled)
* Memory: 32 GB
* Opencore Version: 0.7.5 (not latest version, but should be fine, anyhow, as it is bootloader independent and works with Clover as well)
* MacOS Version: 12.0.1 (also worked on 11.6.2, which I upgraded from)
* SMBIOS: MacPro7,1

### CODE REVIEW

File: AMD/zen.md [hunk 1]
```diff
@@ -127,15 +127,15 @@ Removes device properties from the map, for us we can ignore this
 
 ### Add
 
-Here's where we specify which kexts to load, in what specific order to load, and what architectures each kext is meant for. By default we recommend leaving what ProperTree has done, however for 32-bit CPUs please see below:
+Here's where we specify which kexts to load, in what specific order to load, and what architectures each kext is meant for. Make sure that AMDRyzenCPUPowerManagement comes *before* SMCAMDProcessor if you choose to use them, as SMCAMDProcessor depends on it. Otherwise, we recommend leaving what ProperTree has done by default, however for 32-bit CPUs please see below:
```
Comment (1Revenger1) [inline AMD/zen.md:130]: Does Propertree not get the order correct already? If the kext is defined as needed under the Info.plist, ProperTree should be able to detect this by default.

File: AMD/zen.md [hunk 2]
```diff
@@ -127,15 +127,15 @@ Removes device properties from the map, for us we can ignore this
...
 
 ::: details More in-depth Info
 
 The main thing you need to keep in mind is:
 
 * Load order
   * Remember that any plugins should load *after* its dependencies
-  * This means kexts like Lilu **must** come before VirtualSMC, AppleALC, WhateverGreen, etc
+  * This means kexts like Lilu **must** come before VirtualSMC, AppleALC, WhateverGreen, the kext AMDRyzenCPUPowerManagement before SMCAMDProcessor (if you included them), etc
```
Comment (1Revenger1) [inline AMD/zen.md:138]:
```suggestion
  * This means kexts like Lilu **must** come before VirtualSMC, AppleALC, WhateverGreen, etc. Additionally, AMDRyzenCPUPowerManagement should come before SMCAMDProcessor if you include them.
```

File: ktext.md [hunk 3]
```diff
@@ -308,6 +310,10 @@ However ProperTree will handle this for you, so you need not concern yourself
 * [VoodooHDA](https://sourceforge.net/projects/voodoohda/)
   * Audio for FX systems and front panel Mic+Audio support for Ryzen system, do not mix with AppleALC. Audio quality is noticeably worse than AppleALC on Zen CPUs
   * Requires OS X 10.6 or newer
+* [AMDRyzenCPUPowerManagement](https://github.com/trulyspinach/SMCAMDProcessor/releases/)
+  * Optional CPU temperature monitoring and control for AMD Zen systems, not designed for AMD FX
```
Comment (1Revenger1) [inline ktext.md:314]:
```suggestion
  * Optional CPU temperature monitoring and control for AMD Zen systems. Not designed for AMD FX
```

**Review by 1Revenger1 — CHANGES_REQUESTED** [review]

File: AMD/zen.md (hunk 1 above)
Comment (alyxferrari) [reply AMD/zen.md:130]: ProperTree does get the order correct for SMCAMDProcessor.

**Pull request closed without merging**